librz/core/cdebug.c

```diff
@@ -590,3 +590,46 @@ RZ_API void rz_debug_map_list_visual(RzDebug *dbg, ut64 addr, const char *input,
 		print_debug_maps_ascii_art(dbg, maps, addr, colors);
 	}
 }
+
+RZ_API void rz_debug_trace_list(RzDebug *dbg, RzOutputMode mode, ut64 offset) {
```

Comment:
1. Please add Doxygen
2. Please add `rz_return_if_fail(dbg)`
3. Since it's located in `librz/core` there is no need to use `dbg->cb_printf()` and `rz_cons_printf()` could be used directly instead.

Reply:
I suggest also renaming it from `*_list()` to `*_print()` - we usually use `_list()` suffix for functions that return `RzList`.